Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: all start stop restart logs ps setup-clients setup-consent-store setup show-secrets clean-clients configure-google-auth
+.PHONY: all start start-backend start-frontend stop restart logs ps setup-clients setup-consent-store setup show-secrets clean-clients configure-google-auth
```

checkmake (0.2.2)

[warning] Missing required phony target "clean"

(minphony)